Tests/SWBBuildSystemTests/CustomTaskBuildOperationTests.swift

```diff
@@ -17,13 +17,24 @@ import SWBCore
 import SWBTestSupport
 import SWBTaskExecution
 import SWBUtil
+import SWBProtocol
 
 @Suite
 fileprivate struct CustomTaskBuildOperationTests: CoreBasedTests {
 
     @Test(.requireSDKs(.host))
     func outputParsing() async throws {
         try await withTemporaryDirectory { tmpDir in
+            let destination: RunDestinationInfo = .host
+            let core = try await getCore()
+            let toolchain = try #require(core.toolchainRegistry.defaultToolchain)
+            let environment: [String: String]
+            if  destination.imageFormat(core) == .elf {
+                environment = ["LD_LIBRARY_PATH": toolchain.path.join("usr/lib/swift/\(destination.platform)").str]
+            } else {
+                environment = [:]
```

Comment:
Wonder if Windows needs PATH set to this path as well? Could be a separate change though